skyrimnet-xtts/skyrimnet_xtts.py

```
@@ -54,31 +53,14 @@
-    if (_USE_API_MODE or _FROM_GRADIO) and api_value is not None:
-        return api_value
-    
-    # Otherwise, only return config file value (ignore API value)
-    return config_value
-
-
 # =============================================================================
 # MAIN APPLICATION FUNCTIONS
 # =============================================================================
 
 def generate_audio(model_choice:str=None, text:str=None, language:str="en", speaker_audio:str=None, prefix_audio:str=None,
```

Ruff (0.14.3)

Unused function argument: `model_choice`

(ARG001)